boards/arm/frdm_k64f/doc/frdm_k64f.rst

```diff
@@ -223,6 +220,21 @@ the following message:
 
    Hello World! arm
 
+Debugging
+=========
+
+This example uses the :ref:`hello_world` sample with the
+:ref:`nxp_opensda_pyocd` tools. Use the ``make debug`` build target to build
+your Zephyr application, invoke the pyOCD GDB server, attach a GDB client, and
+program your Zephyr application to flash. It will leave you at a gdb prompt.
+
+.. code-block:: console
+
+   $ cd $ZEPHYR_BASE
+   $ . zephyr-env.sh
```

Comment:
Good point. How do you recommend I fix this? I think I could either 1) change line 233 to ``cd zephyr`` or 2) remove lines 233 and 234 altogether.